public/controllers/management/components/management/status/status-agent-info.js

```diff
@@ -75,14 +75,14 @@ export class WzStatusAgentInfo extends Component {
         </EuiFlexGroup>
         <EuiFlexGroup>
           <EuiFlexItem>Status</EuiFlexItem>
-          <EuiFlexItem style={greyStyle}>{agentInfo.status}</EuiFlexItem>
+          <EuiFlexItem style={greyStyle}>{agentInfo.status.charAt(0).toUpperCase() + status.slice(1)}</EuiFlexItem>
```

Comment:
The same outcome can be achieved with `text-transform: capitalize` style and would prevent the use of javascript to change the Status text.